### PR TITLE
ON-10055 # add FormSubmissionFileAccessToken types

### PR DESCRIPTION
**Pull request by divporter**

### CODE REVIEW

File: flow-typed/submissions.js [hunk 1]
```diff
@@ -178,3 +178,14 @@ declare type GeneratePaymentConfigPayload = {
...
+
+declare type NewFormSubmissionFileAccessToken = {
+  expiry: string,
+  s3Location: string,
+  submisionId: string,
+  emailAddress: string,
+}
+
+declare type FormSubmissionFileAccessToken = {
+  uniqueId: string,
```
Comment (mymattcarroll) [inline flow-typed/submissions.js:190]: Can we just call this `id` like our other types?

File: flow-typed/submissions.js [hunk 2]
```diff
@@ -178,3 +178,14 @@ declare type GeneratePaymentConfigPayload = {
   crn2?: string,
   crn3?: string,
 }
+
+declare type NewFormSubmissionFileAccessToken = {
+  expiry: string,
```
Comment (mymattcarroll) [inline flow-typed/submissions.js:183]: can we rename this to `expiresAt` to match the standard we used for other timestamps

File: flow-typed/submissions.js (hunk 1 above)
Comment (mymattcarroll) [reply flow-typed/submissions.js:190]: Can we also add a `createdAt` timestamp property please.

File: flow-typed/submissions.js [hunk 3]
```diff
@@ -178,3 +178,14 @@ declare type GeneratePaymentConfigPayload = {
...
+  s3Location: string,
```
Comment (mymattcarroll) [inline flow-typed/submissions.js:184]: Should this be
```js
s3: {
  bucket: string,
  key: string
}
```

File: flow-typed/submissions.js [hunk 4]
```diff
@@ -178,3 +178,14 @@ declare type GeneratePaymentConfigPayload = {
...
+  submisionId: string,
+  emailAddress: string,
```
Comment (mymattcarroll) [inline flow-typed/submissions.js:186]: Whats the purpose of storing this?

Comment (divporter) [reply flow-typed/submissions.js:186]: We agreed just for record keeping purposes

**Pull request merged**